### ISSUE TITLE
List of Spectrum Tag Names

### ISSUE THREAD

**Issue by velara3** [adobe/spectrum-web-components#1006]
Is there a constantly updated online list of Spectrum tag names that are linebreak or comma separated? 

Tag names: 
 - sp-accordion
 - sp-accordion-item
 - sp-action-button
 - sp-action-group
 - ...
 - sp-top-nav
 - sp-top-nav-item

File data: 
```
sp-accordion
sp-accordion-item
sp-action-button
sp-action-group
sp-top-nav
sp-top-nav-item
```

My use case is I have a form that I want to have Spectrum component names in a dropdown list or in auto complete. 

https://opensource.adobe.com/spectrum-web-components/

**Comment by Westbrook** [comment]
Hi @velara3, at this time we don't maintain this sort of list, and I don't see that coming up in our roadmap anytime soon. Even as we expand the documentation site to better cover the "sub-elements" of each package, there will likely always been some that don't deserve their own entry, so while we have plans to publish a better sitemap for that content, even that would likely be incomplete. I'd love to hear what you're looking to build here, however.

**Comment by velara3** [comment]
There are a few use cases. One is where you are in a code editor where you want to add code completers like: 

![image](https://user-images.githubusercontent.com/1084127/99022817-f4264380-2528-11eb-8030-42757edddae9.png)

In Ace Editor this is how you add new keywords: 

https://github.com/ajaxorg/ace/blob/ce8eec6b4ed4d485e8bc13ce41ca123b864773ed/lib/ace/ext/language_tools.js#L94

It could also be a json file or tmlanguage. 

Basically, adding `sp-button` and other Spectrum components to code complete when editing HTML. So if you type, `<button` then `button` and `sp-button` both show in code complete. 

I haven't worked in code editors in a while so my phrasing may be off.

**Comment by Westbrook** [comment]
I see. We leverage an early version of https://github.com/webcomponents/custom-elements-manifest via the `custom-element.json` file in each of our packages (e.g. https://unpkg.com/browse/@spectrum-web-components/button@0.9.4/custom-elements.json) to empower tools like https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin to deliver this sort of functionality in editor. It also goes one step further to support available attributes/events/properties and their types as well. Hopefully between the default capabilities of the tools listed here, or via the `custom-elements.json` file in each package, you'll be able to accomplish the goals that you have in this area.
